code/curve25519/Hacl.Curve25519_51.fsti

```diff
@@ -3,6 +3,24 @@ module Hacl.Curve25519_51
 open Hacl.Impl.Curve25519.Generic
 open Hacl.Impl.Curve25519.Fields
 
+[@@ Comment "Compute the scalar multiple of a point.
```

Comment:
Should we add the length of public and secret keys?

Reply:
Added in 2cbd6d0f35d3859338fd736d64c3bc8c8ba47a62.